features/page_objects/choose_exemptions_page.rb

```diff
@@ -10,18 +10,36 @@ class ChooseExemptionsPage < SitePrism::Page
 
   element(:submit_button, "input[name='commit']")
 
-  def submit(args = {})
-    search_val = "checkbox-#{args[:exemption]}"
-
-    select_tab(args[:tab])
-
-    if args.key?(:exemption)
-      exemptions.find { |chk| chk["id"] == search_val }.click
+  # Takes each exemption number in the array
+  # Identifies what tab that exemption is on using the identify_tab method
+  # Selects that tab using the select_tab method
+  # Then clicks on the relevant checkbox for that exemption number
+  def submit(args = { exemptions: %w(S1 D2) })
```

Comment:
ok don't have a strong view on having a default and so happy to revert it back to how it was previously. I think I was just concerned that one test relies on having two exemptions so that they can be deregistered in sequence to then cause a registration deregistration.